src/screens/PluginStore/PluginModal.tsx

```
@@ -326,55 +346,94 @@ const PluginModal: React.FC<IPluginModalProps> = ({
                         <div className={styles.screenshotsContainer}>
                           {details.screenshots.map((src, idx) => (
                             <img
                               key={idx}
                               src={src}
-                              alt={`Screenshot ${idx + 1}`}
+                              alt={t('modal.screenshotAlt', { index: idx + 1 })}
+                              aria-label={t('modal.screenshotAlt', {
+                                index: idx + 1,
+                              })}
                               className={styles.screenshotThumbnail}
+                              role="button"
```

Biome (2.1.2)

[error] The HTML element img is non-interactive and should not have an interactive role.

Replace img with a div or a span.
Unsafe fix: Remove the role attribute.


(lint/a11y/noNoninteractiveElementToInteractiveRole)